conf/script/build-ha-update

```diff
@@ -89,7 +89,13 @@ if [[ -f $consul_kv_dump ]]; then
 fi
 $consul_bin/consul kv export > $consul_kv_dump
 
+sudo pcs cluster standby --all
+
 reset_all
+
+sudo pcs cluster unstandby --all
+sudo pcs resource cleanup
```

Comment:
Do we really need this command? All the resources would have been deleted when we are here, rt?

Reply:
> 
> 
> @VijayThakkar132 
 Do we really need this command? All the resources would have been deleted when we are here, rt?

Reply:
Yes, I think not needed. Let me give a try on the hardware setup to verify how it goes without cleanup.

Reply:
Software update went well without cleanup also. So, removing cluster cleanup in the next patch.